.github/workflows/tpu-tests.yml

```diff
@@ -8,8 +8,10 @@ on:
     types: [opened, reopened, ready_for_review, synchronize]  # added `ready_for_review` since draft is skipped
     paths:
       - ".github/workflows/tpu-tests.yml"
+      - "requirements/lite/**"
+      - "src/lightning_lite/**"
+      - "tests/tests_lite/**"
       - "requirements/pytorch/*"
-      - "!requirements/pytorch/docs.txt"
```

Comment:
I think this is fine as there is no reason to run TPU tests if you just edit docs

Reply:
found answer in https://github.com/Lightning-AI/lightning/pull/15633/files#r1019732071